src/components/SendTokensOne.js

```diff
@@ -84,6 +86,13 @@ class SendTokensOne extends React.Component {
     this.setState({ inputsCount: newCount });
   }
 
+  /**
+   * Use helper method to check if token is an NFT
+   */
+  isNFT = () => {
+    return helpers.isTokenNFT(_.get(this.state, 'selected.uid'), this.props.tokenMetadata);
```

Comment:
The idea of using `_.get` is because it already checks if the key exists in the object. So if I want to get `this.state.selected.uid` but I don't know if `this.state.selected` exists I need to do `this.state.selected && this.state.selected.uid`. With `_.get` I can safely do that and it handles the existence for me.

If `selected` is null it will return `false`, i.e. will continue the same old behaviour of the screen

Reply:
What does `isTokenNFT(...)` expect for the first parameter? Is a `false` an expected type? If yes, I'm good to resolve this conversation.

Reply:
I think I was not clear with the words. If `selected` is `null`, then `isTokenNFT` will return `false`.

If `selected` is `null`,  `_.get` will return `undefined`, then the first thing we do is `uid in metadata` in this case `undefined in metadata`, then returns `false`.

Reply:
I got it. `_.get(...)` will return `undefined`. And `isTokenNFT(undefined, ...)` will return false. Ok, then!